pkg/ansible/watches/watches.go

```diff
@@ -312,3 +297,18 @@ func getAnsibleVerbosity(gvk schema.GroupVersionKind, defValue int) int {
 	}
 	return ansibleVerbosity
 }
+
+// Returns value for MaxWorkers/Ansibleverbosity based on if envVar is set or a defvalue is used.
```

Comment:
Nit: Go doc convention is to use the identifier name (in this case the function name) as the first word of the comment. Not a huge deal since this function is unexported, but a good habit nonetheless.